test/test_sys_checkout.py

```diff
@@ -1591,7 +1681,8 @@ def setUp(self):
         """
 
         # Run the basic setup
-        super(TestSubrepoCheckout, self).setUp()
+        #super(TestSubrepoCheckout, self).setUp()
+        super().setUp()
```

Comment:
Why the comment? In case we need to bring back version 2 support?

Reply:
fixed